config/software/supermarket.rb

```diff
@@ -15,10 +15,11 @@
 #
 
 name "supermarket"
-default_version "master"
+default_version 'master'
```

Comment:
Not sure why you're removing the double quotes (since they are better and you must be a bad person for not liking them :P), but everything else in this file uses double quotes.